pkg/cmd/validate.go

```diff
@@ -18,3 +21,22 @@ func Validate(vs ...cobra.PositionalArgs) cobra.PositionalArgs {
 		return nil
 	}
 }
+
+func CheckStackName(cmd *cobra.Command, args []string) error {
+	if notOk, err := stackNameIsInDnsCharset(args[0]); err != nil || notOk {
```

Comment:
I would add a check here to make sure args has at least one element, otherwise this will panic. The way the error catching logic works, we consume all checks and aggregate the errors so this might potentially trigger a panic if someone runs the command without any args